File: ranker-algorithm/course.py
class Course:
    def __init__(self, department, course, year, semester, prof, gpa, section, a, b, c, d, f, q, finished, enrolled):
        self.department = department
        self.course = course
        self.year = year
        self.semester = semester
        self.prof = prof
        self.gpa = float(gpa)
        self.section = section
        self.a = int(a)
        self.b = int(b)
        self.c = int(c)
        self.d = int(d)
        self.f = int(f)
        self.q = int(q)
        self.finished = int(finished)
        self.enrolled = int(enrolled)

File: course-ranker/src/course.js
class Course {
    constructor (department, course, year, semester, prof, gpa, section, a, b, c, d, f, q, finished, enrolled) {
        this.department = department;
        this.course = course;
        this.year = year;
        this.semester = semester;
        this.prof = prof;
        this.gpa = Number.parseFloat(gpa);
        this.section = section;
        this.a = Number.parseInt(a);
        this.b = Number.parseInt(b);
        this.c = Number.parseInt(c);
        this.d = Number.parseInt(d);
        this.f = Number.parseInt(f);
        this.q = Number.parseInt(q);
        this.finished = Number.parseInt(finished);
        this.enrolled = Number.parseInt(enrolled);
    }

}